_posts/pandera-dataframe-validation.md

```diff
@@ -0,0 +1,263 @@
+---
+layout: single
+title: "Statistical Data Validation with Pandera"
+excerpt: "A flexible and expressive toolkit for performing statistical validation checks on pandas data structures."
+author: "Niels Bantilan"
+permalink: /posts/pandera-dataframe-validation
```

Comment:
```suggestion
permalink: /posts/pandera-python-pandas-dataframe-validation
```

Reply:
google looks at url's. so similar to the title i'd like to see python and pandas there if it makes sense!